### PR TITLE
Update tutorials to .NET 6

### LINKED ISSUE #27896: Following the instructions gives me different results.
1. Using `dotenv new console` doesn't create a file with the results that follow the article text: "... Open Program.cs. It should look like this:"
2. Following the instruction: "Create a new file using the File:New command. Name it BankAccount.cs. Add the following code to your BankAccount.cs file:" creates a file but not in the same directory. Where should we save it?
3. When pasting the code into `Program.cs` I get an error line under `BankAccount` in this line: `var account = new BankAccount("jimmy", 1000);`. The error reads: `CS0246	The type or namespace name 'BankAccount' could not be found (are you missing a using directive or an assembly reference?)`

These are trivial issues if you've some coding experience, but these are aimed at beginners so I would imagine are unusable for many new-starters

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 24c8353f-69e7-9068-9b18-df0198a1b122
* Version Independent ID: 823031fe-c8a4-69cd-2b1a-9faacd282aab
* Content: [Classes and objects  - C# Fundamentals tutorial](https://docs.microsoft.com/en-gb/dotnet/csharp/fundamentals/tutorials/classes)
* Content Source: [docs/csharp/fundamentals/tutorials/classes.md](https://github.com/dotnet/docs/blob/main/docs/csharp/fundamentals/tutorials/classes.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes [#27896](https://github.com/dotnet/docs/issues/27896)

Also, in this PR, update all the tutorials in the Get Started and Tutorials section to .NET 6. That includes using top-level statements, implicit usings, and following other conventions.

Because I changed so much of the included code, I updated the date on the associated articles even though the text didn't change as much.

### CODE REVIEW

File: docs/csharp/fundamentals/tutorials/classes.md [hunk 1]
```diff
@@ -16,18 +16,8 @@ The tutorial expects that you have a machine set up for local development. On Wi
...
-        {
-            Console.WriteLine("Hello World!");
-        }
-    }
-}
+// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
 ```
 
 In this tutorial, you're going to create new types that represent a bank account. Typically developers define each class in a different text file. That makes it easier to manage as a program grows in size. Create a new file named *BankAccount.cs* in the *classes* directory.
```
Comment (IEvangelist) [inline docs/csharp/fundamentals/tutorials/classes.md:23]:
```suggestion
In this tutorial, you're going to create new types that represent a bank account. Typically developers define each class in a different text file. That makes it easier to manage as a program grows in size. Create a new file named *BankAccount.cs* in the *Classes* directory.
```

File: docs/csharp/fundamentals/tutorials/classes.md [hunk 2]
```diff
@@ -89,6 +75,8 @@ public BankAccount(string name, decimal initialBalance)
 Constructors are called when you create an object using [`new`](../../language-reference/operators/new-operator.md). Replace the line `Console.WriteLine("Hello World!");` in *Program.cs* with the following code (replace `<name>` with your name):
 
 ```csharp
+using classes;
```
Comment (IEvangelist) [inline docs/csharp/fundamentals/tutorials/classes.md:78]:
```suggestion
using Classes;
```

File: docs/csharp/fundamentals/tutorials/snippets/introduction-to-classes/transaction.cs [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
-        {
-            this.Amount = amount;
-            this.Date = date;
-            this.Notes = note;
-        }
+    public Transaction(decimal amount, DateTime date, string note)
+    {
+        this.Amount = amount;
+        this.Date = date;
+        this.Notes = note;
```
Comment (IEvangelist) [inline docs/csharp/fundamentals/tutorials/snippets/introduction-to-classes/transaction.cs:12]:
```suggestion
        Amount = amount;
        Date = date;
        Notes = note;
```

File: docs/csharp/fundamentals/tutorials/snippets/patterns/start/toll-calculator/Program.cs [hunk 4]
```diff
@@ -3,126 +3,121 @@
 using ConsumerVehicleRegistration;
 using LiveryRegistration;
 
-namespace toll_calculator
+/*   First set of test code
+using toll_calculator;
```
Comment (IEvangelist) [inline docs/csharp/fundamentals/tutorials/snippets/patterns/start/toll-calculator/Program.cs:7]:
```suggestion
using Calculators;
```

**Review by IEvangelist — APPROVED** [review]
These are awesome updates, Bill! Great job, I love how much this cleans it up. I have a few suggestions for the namespace bits and consistency with naming conventions that I'd like to propose. Other than that, let's ship it.

File: docs/csharp/fundamentals/tutorials/snippets/object-oriented-programming/BankAccount.cs [hunk 5]
```diff
@@ -9,7 +9,7 @@ public decimal Balance
         get
         {
             decimal balance = 0;
-            foreach (var item in allTransactions)
+            foreach (var item in _allTransactions)
             {
                 balance += item.Amount;
             }
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/csharp/fundamentals/tutorials/snippets/object-oriented-programming/BankAccount.cs:15]: I'm waiting. This is a very early tutorial. I'm adding a section that introduces LINQ in the upcoming "fundamentals" section. That will update this tutorial with that functionality.

File: docs/csharp/fundamentals/tutorials/snippets/inheritance/is-a.cs [hunk 6]
```diff
@@ -1,24 +1,22 @@
...
 {
     public Automobile(string make, string model, int year)
     {
         if (make == null)
-           throw new ArgumentNullException(nameof(make), "The make cannot be null.");
-        else if (String.IsNullOrWhiteSpace(make))
-           throw new ArgumentException("make cannot be an empty string or have space characters only.");
+            throw new ArgumentNullException(nameof(make), "The make cannot be null.");
+        else if (string.IsNullOrWhiteSpace(make))
+            throw new ArgumentException("make cannot be an empty string or have space characters only.");
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply docs/csharp/fundamentals/tutorials/snippets/inheritance/is-a.cs:9]: I'm torn. I'm thinking of using the `!!` operator in the next set of updates.

File: docs/csharp/fundamentals/tutorials/snippets/inheritance/inheritance.csproj [hunk 7]
```diff
@@ -2,9 +2,10 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
     <StartupObject>inheritance.Program</StartupObject>
```
Comment (BillWagner) [inline docs/csharp/fundamentals/tutorials/snippets/inheritance/inheritance.csproj:7]:
```suggestion
    <StartupObject>Inheritance.Program</StartupObject>
```

File: docs/csharp/fundamentals/tutorials/snippets/patterns/finished/toll-calculator/Program.cs [hunk 8]
```diff
@@ -1,99 +1,91 @@
...
 using LiveryRegistration;
 
-namespace toll_calculator
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            var tollCalc = new TollCalculator();
+using TollCalculator;
```
Comment (BillWagner) [inline docs/csharp/fundamentals/tutorials/snippets/patterns/finished/toll-calculator/Program.cs:5]:
```suggestion
using Calculators;
```

**Pull request merged**